buildlib/pr/go/go-test.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         ./autogen.sh
         mkdir build
         cd build
-        ../contrib/configure-devel --prefix=$(Build.Repository.LocalPath)/install \
+        ../contrib/configure-devel --prefix=$(Build.ArtifactStagingDirectory)/ucx-install \
```

Comment:
why using ArtifactStagingDirectory? is it for OOS build?

Reply:
Yes for OOS test

Reply:
maybe comment it; when looking at this it seems we're trying create artifacts but it's not really the purpose